src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -618,15 +618,31 @@ public function collectFromCode($content, $fileName, Module $module)
         $potentialClassName = null;
         $currentUse = null;
         $currentUseAlias = null;
+
+        $inVar = null;
+        $stringVariables = [];
         foreach ($tokens as $token) {
             // Shuffle last token to $lastToken
             $previousToken = $thisToken;
             $thisToken = $token;
+
             if (is_array($token)) {
                 list($id, $text, $lineNo) = $token;
                 // minus 2 is used so the the line we get corresponds with what number token_get_all() returned
                 $line = $lines[$lineNo - 2] ?? '';
 
+                // Track string variables
+                if ($id === T_VARIABLE) {
+                    $inVar = $text;
+                }
+                if ($id === T_CONSTANT_ENCAPSED_STRING && $inVar && $text) {
+                    $stringVariables[$inVar] = $text;
+                    $inVar = null;
+                }
+                if ($text === ';') {
```

Comment:
```suggestion
                // Stop tracking the variable as we've finished parsing its assignment
                if ($text === ';') {
```
This assumes my earlier assumption about what `$inVar` is used for is correct.

Reply:
yes